aten/src/ATen/native/DispatchStub.h

```diff
@@ -170,6 +172,10 @@ struct DispatchStub<rT (*)(Args...), T> {
     impl.mps_dispatch_ptr = reinterpret_cast<void*>(fn_ptr);
   }
 
+  void set_private_use1_dispatch_ptr(FnPtr fn_ptr) {
```

Comment:
nit: can you make it `set_privateuse1_dispatch_ptr` (for greppability)

Reply:
done